code/ui/manager/src/components/mobile/navigation/MobileAddonsDrawer.tsx

```diff
@@ -1,59 +1,22 @@
 import type { FC, ReactNode } from 'react';
-import React, { useRef } from 'react';
+import React from 'react';
 import { styled } from '@storybook/theming';
-import { Transition } from 'react-transition-group';
-import type { TransitionStatus } from 'react-transition-group/Transition';
-import { useLayout } from '../../layout/LayoutProvider';
 
 interface MobileAddonsDrawerProps {
   children: ReactNode;
 }
 
-const TRANSITION_DURATION = 200;
-
-const Container = styled.div<{ state: TransitionStatus }>(({ theme, state }) => ({
-  position: 'fixed',
+const Container = styled.div(({ theme }) => ({
   boxSizing: 'border-box',
   width: '100%',
   background: theme.background.content,
-  height: '50%',
-  bottom: 0,
-  left: 0,
+  height: '42vh',
   zIndex: 11,
-  transition: `all ${TRANSITION_DURATION}ms ease-in-out`,
+  transition: 'all 200ms ease-in-out',
   overflow: 'hidden',
-  borderTop: `1px solid ${theme.appBorderColor}`,
-  transform: `${(() => {
-    switch (state) {
-      case 'entering':
-      case 'entered':
-        return 'translateY(0)';
-      case 'exiting':
-      case 'exited':
-        return 'translateY(100%)';
-      default:
-        return 'translateY(0)';
-    }
-  })()}`,
+  transform: 'translateY(0)',
```

Comment:
can this be removed?

Reply:
Yes. I replaced it by `position: 'relative'`